Landing Page

armansito opened the pull request:
Bare bones homepage for the paper. Contains links to arXiv, this GitHub repository, embedded YouTube video of the 40 second reel, and the abstract.

Some things to add in the short term:
- A bit more detail about the code and how to find the shader code in Vello.
- A natbib citation blurb
- Embedded version of https://levien.com/tmp/beztoy/

armansito committed:
Beginnings of the landing page

raphlinus commented:
Definitely looks good as a start. I think the HTML file (and any other assets it links to, including the wasm for the interactive example) should be in a `docs/` subdirectory, otherwise it mixes up serving source. I'm fine with this starting out as static HTML, and *possibly* going to an action to deploy later if we need it, but there's also a pretty good chance we won't.

raphlinus commented on index.html:
```diff
@@ -0,0 +1,121 @@
+  <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+  <style>
+    body {
+      font-family: 'Lato', Verdana, Helvetica, sans-serif;
```
The GF call to bring in Lato is missing, so this renders as Verdana on my system.

armansito replied:
Done.

armansito committed:
embed beztoy

armansito committed:
Fix Lato font; move files to landing_page/

armansito commented:
I moved files relating to the landing page to `landing_page/`. I can rename that to `docs/` too.

I also embedded beztoy with minor modifications. The description for the toy could use a bit more info (e.g. it doesn't explain the red error bar atm). I built the WASM locally (using https://github.com/raphlinus/raphlinus.github.io/pull/100) and simply included the build output here.

raphlinus reviewed (approved):
Looks great! I agree it might be nice to have more description (like the error bar), but I think we can start here.

There's probably a way to use a folder name other than `/docs`, but I can't immediately figure it out. The subdirectories that are there now (`flatten` etc) don't show in the config screen. So I recommend we change the subdirectory to that.

I also like the Lato font well enough :)

armansito committed:
Address comments + tweaks

- Moved the page sources to docs/.
- Bundled a release build of the beztoy JS and WASM.
- Fixed an issue in the interactive toy that caused a control point
  to be permanently lost when dragged out of view.

armansito committed:
Fix indentation
